view.php

```diff
@@ -256,21 +257,42 @@
     $numcolumns = 2;
 
     // Show start/end date or recurring meeting information.
+    $rowmeetingtime = new html_table_row();
+    $rowmeetingtime->id = 'zoom_schedule-meetingtime';
+    $meetingtimeheader = new html_table_cell();
+    $meetingtimeheader->header = true;
+    $meetingtimetext = new html_table_cell();
+
     if ($isrecurringnotime) {
-        $table->data[] = [get_string('recurringmeeting', 'mod_zoom'), get_string('recurringmeetingexplanation', 'mod_zoom')];
+        $meetingtimeheader->text = get_string('recurringmeeting', 'mod_zoom');
+        $meetingtimetext->text = get_string('recurringmeetingexplanation', 'mod_zoom');
     } else if ($zoom->recurring && $zoom->recurrence_type != ZOOM_RECURRINGTYPE_NOTIME) {
-        $table->data[] = [get_string('recurringmeeting', 'mod_zoom'), get_string('recurringmeetingthisis', 'mod_zoom')];
+        $meetingtimeheader->text = get_string('recurringmeeting', 'mod_zoom');
+        $meetingtimetext->text = get_string('recurringmeetingthisis', 'mod_zoom');
```

Comment:
This pair needs to be added to the cells array before the other pair of cells.

Reply:
I just realized it probably needs its own row and cells, so this might be what I was recommending:
```suggestion
        $meetingrecurringheader = new html_table_cell();
        $meetingrecurringheader->header = true;
        $meetingrecurringheader->text = get_string('recurringmeeting', 'mod_zoom');
        $meetingrecurringtext = new html_table_cell();
        $meetingrecurringtext->text = get_string('recurringmeetingthisis', 'mod_zoom');
        $rowmeetingrecurring = new html_table_row();
        $rowmeetingrecurring ->id = 'zoom_schedule-meetingrecurring';
        $rowmeetingrecurring->cells = [$meetingrecurringheader, $meetingrecurringtext];
        $table->data[] = $rowmeetingrecurring;
```

Reply:
I see now. Yeah, that makes sense. Thanks!